src/Mod/Draft/draftmake/make_patharray.py

```diff
@@ -27,10 +27,12 @@
 # \brief This module provides the code for Draft make_path_array function.
 
 import FreeCAD as App
+import FreeCADGui as Gui
```

Comment:
You aren't using the the `FreeCADGui` module in this code, so you should not import it.

Reply:
Done.